ch_backup/ch_backup.py

```diff
@@ -309,7 +309,7 @@ def delete(
         retained_backups = []
         with self._context.locker():
             for i, backup in enumerate(
-                self._context.backup_layout.get_backups(use_light_meta=False)
+                self._context.backup_layout.get_backups(use_light_meta=True)
```

Comment:
Suggest to add a comment describing why it's important to use light metadata here. Something like `# Use light metadata in backups iteration to avoid high memory usage.`.